tests/test_setendings.py

```diff
@@ -0,0 +1,171 @@
+"""
+Tests for setendings.py, check `end_lineno` and `end_col_offset` 
+properties are set.
+
+Astroid Source:
+https://github.com/PyCQA/astroid/blob/master/astroid/transforms.py
+
+See class and method docstrings for explanations.
+
+Run: python tests/test_setendings.py
+"""
+
+from astroid.bases import NodeNG
+# import astroid
+import unittest
+import logging
+from setendings import *
+
+# Set the log level (DEBUG, ERROR, ...), and message format.
+logging.basicConfig(format='', level=logging.DEBUG)
+
+
+class NodeNG(object):
+    """Check `end_col_offset` and `end_lineno` properties are correctly set on
+    the nodes. Use the node method `nodes_of_class`.
+    https://github.com/PyCQA/astroid/blob/master/astroid/node_classes.py#L407
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.reset()
+
+    def reset(self):
+        """Reset between test methods.
+        """
+        # Keep a list rather than a set, because "too many" is wrong.
+        self._props_check = []
+
+    def check_endings(self, module, node_class):
+        for child_node in module.body:
```

Comment:
Maybe I'm missing something here. Is it not possible to call `nodes_of_class` on the `module` already?